### PR TITLE
Unpack parameter integration test

### PR DESCRIPTION
**Pull request by graemecode**
To be fair, this is a effectively a unit test since it's a pure function, but I'm happy to keep it in this directory for redundancy/completeness. I also refactored some of the scenario scaffolding, and added the first basic test for `unpackParametersFromBytes`.

### CODE REVIEW

File: test/ts/integration/examples/simple_interest_terms_contract/runners/unpack_parameters_from_bytes.ts [hunk 1]
```diff
@@ -1,36 +1,37 @@
...
-//                 ));
-//                expect()
-//             });
-//         });
-//     }
-// }
+// External libraries
+import { expect } from "chai";
+
+// Scenario runners
```
Comment (nadavhollander) [inline test/ts/integration/examples/simple_interest_terms_contract/runners/unpack_parameters_from_bytes.ts]: Nit: `Scenario runners` => `Scenarios`

File: test/ts/integration/examples/simple_interest_terms_contract/scenarios/unpack_parameters_from_bytes.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        termLengthUnits: new BigNumber(4),
+    },
+};
+
+export const UNPACK_PARAMETERS_FROM_BYTES_SCENARIOS: UnpackParametersFromBytesScenario[] = [
+    {
+        description: "when given '0x00000000000de0b6b3a76400000009c410004000000000000000000000000000'",
+        ...defaultArgs,
+        input: "0x00000000000de0b6b3a76400000009c410004000000000000000000000000000",
+    },
```
Comment (nadavhollander) [inline test/ts/integration/examples/simple_interest_terms_contract/scenarios/unpack_parameters_from_bytes.ts]: Would be great to add at least 2 more scenarios!

**Comment by graemecode** [comment]
♻️  @nadavhollander see latest commit

**Comment by nadavhollander** [comment]
LGTM!  💃

**Pull request merged**